src/defaults/defaults.go

```diff
@@ -15,30 +18,10 @@ type platformDefaultParameters struct {
 	DefaultConfigFile string
 
 	// Multicast interfaces
-	DefaultMulticastInterfaces []MulticastInterfaceConfig
+	DefaultMulticastInterfaces []string
```

Comment:
It looks like `MulticastInterfaceConfig` is no longer being used? I think that would break existing configs and prevent nodes from being able to configure some options that are available in the current setup (separately controlling if the node listens and/or beacons on an interface, and controlling which port it listens on).

Reply:
Ah, oops, you're right. I'll fix this in a bit.